Cargo.toml

```diff
@@ -8,6 +8,7 @@ edition = "2018"
 adblock = { version = "~0.3.15", default-features = false, features = ["full-regex-handling", "object-pooling"] }
 serde_json = "1.0"
 libc = "0.2"
+regex = "1.5.5"
```

Comment:
Thanks for the contribution, but this shouldn't be added to `Cargo.toml` because we're not using it as a direct dependency. Just the `Cargo.lock` change should be sufficient.